api/envoy/extensions/wasm/v3/wasm.proto

```diff
@@ -79,7 +79,7 @@ message VmConfig {
   // This runtime is not enabled in the official build.
   //
   // [#extension-category: envoy.wasm.runtime]
-  string runtime = 2 [(validate.rules).string = {min_len: 1}];
+  string runtime = 2;
```

Comment:
should this be an enum?

Reply:
I think we can make this as an enum. But the idea of this PR is to make wasm runtime optional because currently envoy only supports one wasm engine (which is the engine used at envoy build-time) at a time. So making this enum or not makes not much difference? WDYT?

Reply:
I think we can make this as an enum. But the idea of this PR is to make wasm runtime optional because currently envoy only supports one wasm engine (which is the engine used at envoy build-time) at a time. So making this enum or not makes not much difference? WDYT?

Reply:
No we shouldn't change existing API and this is registered as extension so should be as is.